### DISCUSSION TITLE
System error: Null every time I try to start a broadcast

### DISCUSSION THREAD

**Discussion by Avillabrille** [ant-media/Ant-Media-Server#6256] [Q&A]
Im pretty sure all the steps are fine but it wont start http://96.126.116.136:5080/LiveApp/streams/4oBDSpUz6fTV1712088247659.mp4

**Comment by yashtandon113** [comment]
@Avillabrille 
Thanks for raising the issue.

May I know which version of AMS you are using and is it possible to share your AMS web panel credentials with my email address, yash.tandon@antmedia.io so I can check further?

Please update here when you share.
